src/main/java/lotto/view/Input.java

```diff
@@ -0,0 +1,62 @@
+package lotto.view;
+
+import lotto.domain.Lotto;
+import java.util.Arrays;
+import java.util.List;
+
+import static camp.nextstep.edu.missionutils.Console.readLine;
+import static lotto.validate.ValidateInput.validateAmount;
+import static lotto.validate.ValidateInput.validateBonusNumber;
+import static lotto.validate.ValidateInput.validateWinningNumbers;
+
+public class Input {
+    public int inputPurchaseAmount(){
+        while (true) {
+            try {
+                System.out.println("구입금액을 입력해 주세요.");
+                String inputAmount = readLine();
+
+                return validateAmount(inputAmount);
+            } catch (IllegalArgumentException e) {
+                System.out.println(e.getMessage());
+            }
+        }
```

Comment:
저는 inputview에서는 값 반환만 하고 다른 곳에서 예외일 경우에 반복처리를 했는데 정은님 처럼 이렇게 하면 이 메서드가 최종적으로 유효한 입력을 반환하는 장점이 있겠네요!

Reply:
```suggestion
    public int inputPurchaseAmount(){
        try {
            System.out.println("구입금액을 입력해 주세요.");
            String inputAmount = readLine();

            return validateAmount(inputAmount);
        } catch (IllegalArgumentException e) {
            System.out.println(e.getMessage());
            
            return inputPurchaseAmount();
        }
```

이렇게 하면 while문을 안 써도 되긴 합니다!